### PR TITLE
Tuiview movetogithub

### PR DESCRIPTION
**Pull request by neilflood**

### CODE REVIEW

File: var/spack/repos/builtin/packages/py-tuiview/package.py [hunk 1]
```diff
@@ -11,11 +11,13 @@ class PyTuiview(PythonPackage):
     table manipulation abilities.
     """
 
-    homepage = "https://bitbucket.org/chchrsc/tuiview"
-    url      = "https://bitbucket.org/chchrsc/tuiview/get/tuiview-1.1.7.tar.gz"
+    homepage = "https://github.com/ubarsc/tuiview"
+    url      = "https://github.com/ubarsc/tuiview/releases/download/tuiview-1.2.6/tuiview-1.2.6.tar.gz"
 
+    version('1.2.6', sha256='61b136fa31c949d7a7a4dbf8562e6fc677d5b1845b152ec39e337f4eb2e91662')
     version('1.1.7', sha256='fbf0bf29cc775357dad4f8a2f0c2ffa98bbf69d603a96353e75b321adef67573')
```
Comment (scheibelp) [inline var/spack/repos/builtin/packages/py-tuiview/package.py]: It looks like this version is no longer available (at the new link or the old one): can you remove it?

Comment (neilflood) [reply var/spack/repos/builtin/packages/py-tuiview/package.py]: Both those versions are available the github site. What went wrong for you?
This works, too
    wget https://github.com/ubarsc/tuiview/releases/download/tuiview-1.1.7/tuiview-1.1.7.tar.gz

I can remove it if you like, but I think in general the idea is to keep old versions where possible.

Comment (neilflood) [reply var/spack/repos/builtin/packages/py-tuiview/package.py]: Both those versions are available the github site. What went wrong for you?
This works, too
wget https://github.com/ubarsc/tuiview/releases/download/tuiview-1.1.7/tuiview-1.1.7.tar.gz

I can remove it if you like, but I think in general the idea is to keep old versions where possible.

(I think I replied in the wrong place, sorry if that comes out twice)

File: var/spack/repos/builtin/packages/py-tuiview/package.py [hunk 2]
```diff
@@ -11,11 +11,13 @@ class PyTuiview(PythonPackage):
...
 
-    depends_on("py-pyqt4", type=('build', 'run'))
+    depends_on("py-pyqt4", type=('build', 'run'), when='@:1.1.99')
+    depends_on("py-pyqt5", type=('build', 'run'), when='@1.2.0:')
     depends_on("py-numpy", type=('build', 'run'))
-    depends_on("gdal")
+    depends_on("gdal+python")
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-tuiview/package.py]:
```suggestion
    depends_on("gdal@1.11.0:+python")
```

**Pull request merged**